example/web-access-control/protected-operation/debug-indirect-acl.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Test ACL in applyMembers mode
```

Comment:
The current ACR draft does not use applyMembers. Instead, it uses memberAccessControl

https://solid.github.io/authorization-panel/acp-specification/#inheritance

Reply:
As discussed, the final changes to ACR will be supported in a separate update once they are implemented such that the harness can be tested.